frontend/src/components/ConfigurationP1.tsx

```diff
@@ -135,7 +145,9 @@ export function Controller(){
   // Gets config data on page load
   useEffect(() => {
     const getConfigData = () => {
-      axios.get(API_CONFIGURATION_URL)
+      axios.get(API_CONFIGURATION_URL,
+        { params: {token: document.cookie, SaaSquatchTenantAlias: config.saasquatchTenantAlias} }
```

Comment:
I realized you can safely remove token: document.cookie now I changed the way this works with the fix!